uabot.code-snippets

```diff
@@ -0,0 +1,741 @@
+{
```

Comment:
What is this file format and how will it display?

Reply:
This is JSON. This will not display anywhere. This is used by Visual Studio Code to add snippet suggestions as you type.

Reply:
Oh duh : facepalm: .

Okay let me review them then haha